### PR TITLE
Add cloud to SystemInfo

### PR DESCRIPTION
**Pull request by mpolden**
@bratseth

### CODE REVIEW

File: hosted-zone-api/src/main/java/ai/vespa/cloud/Cloud.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package ai.vespa.cloud;
+
+/**
+ * The cloud provider of a Vespa Cloud instance.
```
Comment (bratseth) [inline hosted-zone-api/src/main/java/ai/vespa/cloud/Cloud.java:4]:
```suggestion
 * The cloud provider in which a cloud deployment may be running.
 This is "aws" when this runs in Amazon Web Services, and "gcp" when this runs in Google Cloud Platform.
```

File: hosted-zone-api/src/main/java/ai/vespa/cloud/SystemInfo.java [hunk 2]
```diff
@@ -13,12 +13,14 @@ public class SystemInfo {
 
     private final ApplicationId application;
     private final Zone zone;
+    private final Cloud cloud;
     private final Cluster cluster;
     private final Node node;
 
-    public SystemInfo(ApplicationId application, Zone zone, Cluster cluster, Node node) {
```
Comment (bratseth) [inline hosted-zone-api/src/main/java/ai/vespa/cloud/SystemInfo.java:19]: Public API so we must keep the old constructor.

**Review by bratseth — CHANGES_REQUESTED** [review]
Otherwise LGTM

**Comment by mpolden** [comment]
PTAL

**Review by bratseth — APPROVED** [review]
👍

**Pull request merged**